projects/power_distribution/inc/smoketests_pd.h

```diff
@@ -8,6 +8,7 @@
 #define PD_SMOKE_TEST SMOKE_TEST_CURRENT_MEASUREMENT
 
 typedef enum {
+  SMOKE_TEST_UV_CUTOFF = 0,            // see smoke_uv_cutoff.c
   SMOKE_TEST_CURRENT_MEASUREMENT = 0,  // see smoke_current_measurement.c
```

Comment:
It won't work having `= 0` on both - how C enums work is that they're numbered with sequential integers starting at 0, unless you put an `= N` (where `N` is some number), in which case that enum member gets that value. Our convention is that we explicitly set the first value to 0, then let C auto-number the rest of them. So the first value should have `= 0` and the rest shouldn't have anything.